website_critical_css/static/src/xml/page_properties.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<template>
+    <t t-extend="website.pagesMenu.page_info">
+        <t t-jquery="ul li:last-child" t-operation="after">
+            <li class="nav-item">
+                <a
+                    aria-controls="critical_css"
+                    role="tab"
+                    data-toggle="tab"
+                    class="nav-link"
+                    href="#critical_css_tab"
+                >Critical CSS</a>
+            </li>
+        </t>
+        <t t-jquery="#advances_page_info" t-operation="after">
+            <div role="tabpanel" id="critical_css_tab" class="tab-pane fade">
+                <div class="form-group row">
+                    <textarea
+                        class="form-control"
+                        id="critical_css"
+                        style="min-height: 200px"
+                    >
+                        <t t-if="widget.page.critical_css">
+                            <t t-esc="widget.page.critical_css" />
```

Comment:
You can shorten this by using `widget.page.critical_css or ''`

There is always some spaces / tabs in the textarea:
![image](https://user-images.githubusercontent.com/10028499/143461746-869028c0-ef99-4465-94bf-29c5306c3c25.png)
If you save it, it will always enable critical css (unless you manually remove the spaces everytime you want to change something like Page Name). You need to strip them when saving the page properties, or do a check in Python.

Reply:
Great comments, fixed and tested both.